tests/test_ote_training.py

```diff
@@ -0,0 +1,359 @@
+                                                       OTETestExportEvaluationAction,
+                                                       OTETestPotAction,
+                                                       OTETestPotEvaluationAction,
+                                                       OTETestNNCFAction,
+                                                       OTETestNNCFEvaluationAction,
+                                                       OTETestNNCFExportAction,
+                                                       OTETestNNCFExportEvaluationAction)
+
+
+logger = logging.getLogger(__name__)
+
+def DATASET_PARAMETERS_FIELDS() -> List[str]:
+    return deepcopy(['annotations_train',
+                     'images_train_dir',
+                     'annotations_val',
+                     'images_val_dir',
+                     'annotations_test',
+                     'images_test_dir',
+                     ])
+
+DatasetParameters = namedtuple('DatasetParameters', DATASET_PARAMETERS_FIELDS())
+
+
+def _get_dataset_params_from_dataset_definitions(dataset_definitions, dataset_name):
+    if dataset_name not in dataset_definitions:
+        raise ValueError(f'dataset {dataset_name} is absent in dataset_definitions, '
+                         f'dataset_definitions.keys={list(dataset_definitions.keys())}')
+    cur_dataset_definition = dataset_definitions[dataset_name]
+    training_parameters_fields = {k: v for k, v in cur_dataset_definition.items()
+                                  if k in DATASET_PARAMETERS_FIELDS()}
+    make_paths_be_abs(training_parameters_fields, dataset_definitions[ROOT_PATH_KEY])
+
+    assert set(DATASET_PARAMETERS_FIELDS()) == set(training_parameters_fields.keys()), \
+            f'ERROR: dataset definitions for name={dataset_name} does not contain all required fields'
+    assert all(training_parameters_fields.values()), \
+            f'ERROR: dataset definitions for name={dataset_name} contains empty values for some required fields'
+
+    params = DatasetParameters(**training_parameters_fields)
+    return params
+
+def _create_classification_dataset_and_labels_schema(dataset_params):
+    logger.debug(f'Using for train annotation file {dataset_params.annotations_train}')
+    logger.debug(f'Using for val annotation file {dataset_params.annotations_val}')
+    dataset = ClassificationDatasetAdapter(
+        train_data_root=osp.join(dataset_params.images_train_dir),
+        train_ann_file=osp.join(dataset_params.annotations_train),
+        val_data_root=osp.join(dataset_params.images_val_dir),
+        val_ann_file=osp.join(dataset_params.annotations_val),
+        test_data_root=osp.join(dataset_params.images_test_dir),
+        test_ann_file=osp.join(dataset_params.annotations_test))
+    labels_schema = generate_label_schema(dataset.get_labels(), dataset.is_multilabel())
+    return dataset, labels_schema
+
+def get_default_test_action_classes() -> List[Type[BaseOTETestAction]]:
```

Comment:
Please, change the function name: not default -- `image_classification`
And replace below where the function is called